htlcswitch/failure.go

```diff
@@ -30,10 +30,14 @@ type ForwardingError struct {
 // returned.
 func (f *ForwardingError) Error() string {
```

Comment:
Had a quick look at the places where we string match errors in the codebase, and it doesn't look like we do it for ForwardingErrs except the test you've already picked up. May be worth triple checking before merge juust in case.

Reply:
I assume you mean string matching in non-test code? (In test code, the test would fail?) I think that may be quite difficult to triple check, because they may be passed anywhere as type `error`. At this point, I think I trust that everything that really matters doesn't do string matching in `lnd` :grimacing:

Reply:
haha cool, I got nervous just seeing it 😂